rpm/centos-7/Dockerfile.x86_64

```diff
@@ -18,6 +18,7 @@ RUN yum install -y \
    vim-common
 
 ENV GO_VERSION 1.7.5
+ENV DISTRO centos
```

Comment:
can you split the `DISTRO` line changes in the dockerfiles into a separate PR so they can be merged in first?